tools/mom/spec/translate-cluster.ts

```diff
@@ -205,7 +205,10 @@ function translateMetadata(definition: ClusterReference, children: Array<Cluster
 
 // For some reason, "default" fabric access appears in an informational row
 // instead of the access column in many of the core definitions.  Fix this.
-function applyAccessNotes(fields?: HtmlReference, records?: { access?: string }[]) {
+//
+// We also use the presence of this record to add the implicity FabrixIndex
```

Comment:
implicity = typo I think.  Probably mine but you own it now. :)

Reply:
fixed